compose-builder/Makefile

```diff
@@ -537,6 +557,8 @@ down:
 		-f add-device-virtual.yml \
 		-f add-asc-http-export.yml \
 		-f add-asc-mqtt-export.yml \
+		-f add-asc-sample.yml \
+		-f add-app-rfid-llrp-inventory.yml \
```

Comment:
why add `rfid-llrp-inventory` here? is this also fixing a missing bug?

Reply:
yep, it will missing so add it here.